pkg/services/ngalert/notifier/channels/discord_test.go

```diff
@@ -0,0 +1,172 @@
+package channels
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/url"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/services/alerting"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/prometheus/alertmanager/notify"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/prometheus/alertmanager/template"
+	"github.com/prometheus/alertmanager/types"
+	"github.com/prometheus/common/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDiscordNotifier(t *testing.T) {
+	tmpl, err := template.FromGlobs("templates/default.tmpl")
```

Comment:
Could you rebase the PR? This has changed since this and we are no longer keeping default template ready on disk